deploy/openshift/clusterrole.yaml

```diff
@@ -74,6 +74,15 @@ rules:
   - list
   - patch
   - watch
+- apiGroups: ["authentication.istio.io"]
```

Comment:
cc @jmazzitelli @gbaufake same here

Reply:
we need to inform Maistra team about this change. a MAISTRA JIRA would probably be best.